bloom-housing/bloom
# fix: cypress tests working again

# Pull Request Template

## Description
This should make it so the cypress tests pass on core again

## How Can This Be Tested/Reviewed?
you can pull this down and run the various test suites, or just check for the green checks down below

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
